addons/website/models/res_lang.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models, _
+from odoo.exceptions import UserError
+
+
+class Lang(models.Model):
+    _inherit = "res.lang"
+
+    @api.multi
+    def write(self, vals):
+        if vals.get('active') == False:
+            if self.env['website'].search([('language_ids', 'in', self.ids)]):
+                raise UserError(_("Cannot deactivate a language that is currently used by users or websites."))
```

Comment:
Cannot deactivate a language that is currently used on a website

=> User will be checked in the call of super

Reply:
done